src/functions/bench.ts

```diff
@@ -28,14 +30,23 @@ const { getInternals, wrap } = require('../helpers/fnFormat');
  * @param {Array=} opts.cliArgs
  * Array of args to pass to the file itself
  */
-module.exports = async (func, args = [], opts = {}) => {
+export default async (
+  // eslint-disable-next-line @typescript-eslint/ban-types
+  func: Function | (() => null),
+  args: Array<unknown> = [],
```

Comment:
Just describe the form of the function as an arrow function, IE `(x: number) => string` or use `unknown` if we don't need to know what it does.